### DISCUSSION TITLE
Lodestar Standup 2024-37 (September 24, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#7096] [Standups]
### Meeting Info
[September 24, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/sep-24-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- Libp2p offsite update
- Open Discussion
- Team Updates
- Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by philknows** [comment]
Would love to get a formal update on the Electra updates for [alpha.6](https://github.com/ethereum/consensus-specs/releases/tag/v1.5.0-alpha.6)

**Reply by ensi321** [reply]
Related PR: #7094 #7071

**Comment by philknows** [comment]
I'd also like to bring up the idea of experimenting with a [new discord forum channel](https://support.discord.com/hc/en-us/articles/6208479917079-Forum-Channels-FAQ) for `lodestar-help` to better organize and recall solutions for troubleshooting and/or developer type topics to improve persisting + searching for information.

**Comment by philknows** [comment]
We should discuss listening on IPv6 by default and if there's any blockers to doing so. See https://ipv6eth.info/

**Reply by nflaig** [reply]
we have two open issues related to that https://github.com/ChainSafe/lodestar/issues/6532 and https://github.com/ChainSafe/lodestar/issues/6531, not sure if those are a blocker though

**Comment by philknows** [comment]
Please also read up and bring opinions to https://github.com/ChainSafe/lodestar/issues/7098

**Comment by twoeths** [comment]
need to discuss the plan to offload shuffling computation to worker thread or napi-rs asynchronously
more context is [here](https://github.com/ChainSafe/lodestar/issues/6268#issuecomment-2369971927)

**Comment by twoeths** [comment]
I'd like to discuss bringing n-historical states to production, it's been tested for more than 3 months
especially with this PR https://github.com/ChainSafe/lodestar/pull/7104 we'll never have to reload states as analyzed there

**Comment by philknows** [comment]
Closing for #7115 . Notes posted in [usual wiki spot](https://github.com/ChainSafe/lodestar/wiki/Lodestar-Planning-&-Standup-Meetings#september-24-2024-planning-and-standup-meeting).
